### DISCUSSION TITLE
My little wish

### DISCUSSION THREAD

**Discussion by zzz6519003** [castwide/solargraph#689] [Q&A]
I want to be able to jump to the method definish, let's say
```
puts "foo"
```
I click puts, and jump to the definition of puts and see how it's implemented

**Comment by castwide** [comment]
Jump to definition works for most methods, but not for core methods (including `puts`) and some parts of the stdlib. This is because Solargraph uses RBS to generate core maps instead of mapping the core code itself.

This limitation generally applies to methods defined in C extensions as well.
